components/ai_chat/common/mojom/ai_chat.mojom

```diff
@@ -95,11 +97,13 @@ interface PageHandler {
   GetSiteInfo() => (SiteInfo? site_info);
   GetFaviconImageData() => (array<uint8>? favicon_image_data);
   OpenBraveLeoSettings();
-  OpenBraveLeoWiki();
+  OpenURL(url.mojom.Url url);
   DisconnectPageContents();
   ClearConversationHistory();
   RetryAPIRequest();
   GetAPIResponseError() => (APIError error);
+  LikeMessage(bool is_liked, string turn_text) => (string? id);
```

Comment:
Perhaps name this `RateMessage` as that's terminology you've used in the frontend and this is used for both like and dislike?

Reply:
makes sense